docs/benchmarks/micro-benchmarks.md

```diff
@@ -26,18 +26,14 @@ id: micro-benchmarks
         <b>Micro Benchmark</b>
         <img src={require('../assets/bar.png').default}/>
       </td>
-      <td>
-        <b>Model Benchmark</b>
-        <img src={require('../assets/bar.png').default}/>
-      </td>
     </tr>
     <tr valign="top">
       <td align="center" valign="middle">
         <b>Metrics</b>
```

Comment:
already mentioned in table head, can be removed here

Reply:
removed